client/ayon_core/addon/interfaces.py

```diff
@@ -378,18 +424,31 @@ class IHostAddon(AYONInterface):
 
     @property
     @abstractmethod
-    def host_name(self):
+    def host_name(self) -> str:
         """Name of host which addon represents."""
+        raise NotImplementedError
```

Comment:
??? you're raising class....
```suggestion
        pass
```